php/component/field/class-fieldabstract.php

```diff
@@ -270,6 +270,25 @@ public function get_base_name() {
 		return $slug;
 	}
 
+	/**
+	 * Parse slug to label.
+	 *
+	 * @return string
+	 */
+	public function get_input_label() {
+		$label = explode( ' ', str_replace( '_', ' ', $this->get_base_name() ) );
+
+		// Implement Pascal Casing.
+		$label_uc = array_map(
+			function( $label ) {
+				return ucfirst( $label );
+			},
+			$label
+		);
+
+		return implode( ' ', $label_uc );
```

Comment:
Not technically PascalCasing, but it is in the expected format.  Title Casing is more what this is.

Reply:
oops. got it! will update the comment!